### PR TITLE
Adds accordion component

### PR DESCRIPTION
**Pull request by jlhogan**
Also, tweaks some svgs to fix some render issues

### CODE REVIEW

File: src/assets/images/plusCircle.svg [hunk 1]
```diff
@@ -1,4 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18" overflow="visible">
-  <circle cx="50%" cy="50%" fill="currentColor" r="50%"></circle>
-  <text x="50%" y="50%" fill="#fff" style="font-weight:bold;font-size:18px;" text-anchor="middle" alignment-baseline="central">+</text>
-</svg>
\ No newline at end of file
+<svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="plus-circle" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm144 276c0 6.6-5.4 12-12 12h-92v92c0 6.6-5.4 12-12 12h-56c-6.6 0-12-5.4-12-12v-92h-92c-6.6 0-12-5.4-12-12v-56c0-6.6 5.4-12 12-12h92v-92c0-6.6 5.4-12 12-12h56c6.6 0 12 5.4 12 12v92h92c6.6 0 12 5.4 12 12v56z"></path></svg>
```
Comment (mmazanec22) [inline src/assets/images/plusCircle.svg:1]: Should this one also have xmlnsXlink?

File: src/components/Accordion/Accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	titleHeadingLevel: HeadingLevel;
+}
+
+export const Accordion: React.FC<AccordionProps> = ({
+	items,
+	titleHeadingLevel
+}) => {
+	return (
+		<div
+			className="oec-accordion"
```
Comment (mmazanec22) [inline src/components/Accordion/Accordion.tsx:18]: Why not use usa-accordion?

File: src/components/Accordion/AccordionItem.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	collapseText,
+	content,
+	isExpanded: initialIsExpanded = false,
+	headingLevel,
+}) => {
+	const [isExpanded, setIsExpanded] = useState(initialIsExpanded);
+	const Heading = headingLevel;
+	return (
+		<>
+			<Heading className="oec-accordion__heading">
```
Comment (mmazanec22) [inline src/components/Accordion/AccordionItem.tsx:32]: Same comment here-- why not use the USWDS accordion?  https://designsystem.digital.gov/components/accordion/

File: src/components/Accordion/AccordionItem.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+				<button
+					className="oec-accordion__button"
+					onClick={() => setIsExpanded((e) => !e)}
+					aria-expanded={isExpanded}
```
Comment (mmazanec22) [inline src/components/Accordion/AccordionItem.tsx:36]: Yay!

File: src/components/Accordion/Accordion.tsx (hunk 2 above)
Comment (jlhogan) [reply src/components/Accordion/Accordion.tsx:18]: TLDR; it was annoying to figure out the css to override some things they do that we don't want 
usa-accordion uses background image for the expand/collapse icon. If we do that, then i can't rotate the inline svg to rotate the image between exand/collapse state. For some reason, overriding the background image within the existing styles was proving hard for me, and at a certain point it didn't feel worth it to keep trying to do that.

File: src/assets/images/plusCircle.svg (hunk 1 above)
Comment (jlhogan) [reply src/assets/images/plusCircle.svg:1]: no clue what that is! but will add it if it seems useful?

Comment (mmazanec22) [reply src/assets/images/plusCircle.svg:1]: Haha I don't really know either, was just looking at the other svgs and noticed a difference

**Review by mmazanec22 — APPROVED** [review]

**Pull request merged**